hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestOmMetadataManager.java

```diff
@@ -55,6 +56,32 @@ public void setup() throws Exception {
         folder.getRoot().getAbsolutePath());
     omMetadataManager = new OmMetadataManagerImpl(ozoneConfiguration);
   }
+  @Test
+  public void testListVolumes() throws Exception {
+    OmVolumeArgs omVolumeArgs = OmVolumeArgs.newBuilder().setAdminName("admin")
+        .setOwnerName("ownerA").setVolume("volume").build();
+    for (int i = 0; i < 50; i++) {
+      String volName = "vol" + i;
+      String ownerName = "owner" + i;
+      omVolumeArgs = OmVolumeArgs.newBuilder()
+          .setAdminName("admin")
```

Comment:
Nit: Instead of creating a new builder in each iteration, a common builder could be created outside the loop, with common values (`adminName`) already initialized.  Then in each iteration set the specific values and call `build()`.